tests/integration/test_fim/test_files/test_audit/test_audit_multiple_keys.py

```diff
@@ -0,0 +1,160 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+import os
+
+import pytest
+import wazuh_testing.fim as fim
+
+from wazuh_testing import global_parameters
+from wazuh_testing.tools import PREFIX
+from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test, write_wazuh_conf
+from wazuh_testing.tools.file import truncate_file
+from wazuh_testing.tools.configuration import get_wazuh_conf, set_section_wazuh_conf
+from wazuh_testing.tools.services import control_service, get_service
+import wazuh_testing.tools.monitoring as monitoring
+from wazuh_testing.tools import LOG_FILE_PATH
```

Comment:
We should organize the imports:

```suggestion
import os

import pytest

import wazuh_testing.fim as fim
import wazuh_testing.tools.monitoring as monitoring
from wazuh_testing import global_parameters
from wazuh_testing.tools import LOG_FILE_PATH, PREFIX
from wazuh_testing.tools.configuration import check_apply_test, get_wazuh_conf, load_wazuh_configurations, set_section_wazuh_conf,  write_wazuh_conf
from wazuh_testing.tools.file import truncate_file
from wazuh_testing.tools.services import control_service, get_service
```